lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -776,6 +776,16 @@ bool PrimUpdaterManager::mergeToUsd(
     if (!isCopy && TF_VERIFY(ufeMayaItem))
         scene.notify(Ufe::ObjectPreDelete(ufeMayaItem));
 
+    if (!isCopy) {
+        FunctionUndoItem::execute(
```

Comment:
I don't understand this change...why are you putting it outside of undo block? How is it related to UI defaults?

Reply:
I t has its own undo/redo mechanism (calling removeExclude / addExclude) so it does not need to be in the undo block.

I moved it before the merge becuase it worries me that we are merging data with a unactivated prim. Currently it does not seem to affect SdfCopySpecs nor our current diff code, but it may in the future, so Iit seemes ore natural to me to reactivate the prim before trying to merge data in it.

Reply:
You are assuming that this method doesn't need the undo block. This is true today, doesn't have to be true tomorrow. If we don't have an actual issue with the code (reproduced with unit test would be ideal), then let's not do it.
If you can write a unit test to showcase an issue, then let's make this change but in a separate PR.